tests/expose_test.go

```diff
@@ -491,3 +494,40 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		})
 	})
 })
+
+func getNodeHostname(nodeAddresses []k8sv1.NodeAddress) *string {
+	for _, address := range nodeAddresses {
+		if address.Type == k8sv1.NodeHostName {
+			return &address.Address
+		}
+	}
+	return nil
+}
+
+func resolveNodeIp(virtclient kubecli.KubevirtClient, pod *k8sv1.Pod, hostname string, ipFamily k8sv1.IPFamily) (string, error) {
+	output, err := tests.ExecuteCommandOnPod(
+		virtclient,
+		pod,
+		"compute",
+		[]string{"getent", "ahosts", hostname})
+
+	if err != nil {
+		return "", err
+	}
+	for _, ipStr := range strings.Split(output, "\n") {
+		ip := strings.Split(ipStr, " ")[0]
+		if libnet.GetFamily(ip) == ipFamily {
```

Comment:
ahostsv4/ahostsv6 can be used instead.

Reply:
Alright ! Good catch, my ignorant past self was unaware that existed :) 

Thanks.